spec/models/concerns/iiif_print/iiif_print_behavior_spec.rb

```diff
@@ -1,17 +1,21 @@
 require 'spec_helper'
 RSpec.describe IiifPrint::IiifPrintBehavior do
   describe "including_this_module" do
-    before do
-      class PrintWork < ActiveFedora::Base
-        include IiifPrint::IiifPrintBehavior
-      end
-    end
-    let(:klass) { Class.new }
-    subject { PrintWork.new }
+    # before do
+    #   class PrintWork < ActiveFedora::Base
+    #     include IiifPrint::IiifPrintBehavior
+    #   end
+
+    #   class PrintWorkIndexer < Hyrax::WorkIndexer
+    #     include IiifPrint::IiifPrintBehavior
+    #   end
+    # end
+    # let(:klass) { Class.new }
+    # subject { PrintWork.new }
 
-    describe 'split_pdf' do
-      it 'is true' do
-        expect(subject.split_pdf).to be true
+    # TODO: add specs
```

Comment:
and here, or maybe just a big ticket for adding all the specs to these skipped ones